src/components/header/Header.js

```diff
@@ -1,22 +1,58 @@
-import React from 'react';
-
-import { AppBar, Toolbar, Typography } from '@mui/material';
+import cn from 'classnames'
+import { useContext } from 'react';
+import React, { useEffect, useState } from 'react';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { AppBar, Button, Toolbar, Typography } from '@mui/material';
 import logo from '../../images/Logo.svg';
-
-import './style.css';
+import style from './Header.module.scss';
+import logoutIcon from '../../images/logout.icon.svg'
+import { RequestContext } from '../../context/request.context';
 
 const Header = () => {
+    const { requests } = useContext(RequestContext)
+    const [authorized, setAuthorized] = useState(false)
+    const navigate = useNavigate()
+
+    const logout = () => {
+        localStorage.removeItem('token')
+        navigate('/admin/sign-in')
+    }
+
+    useEffect(() => { if (localStorage.getItem('token')) setAuthorized(true) }, [])
```

Comment:
that's to complicated to read.
```suggestion
    useEffect(() => {
       if (localStorage.getItem('token')) setAuthorized(true);
     }, [])
```